src/components/node/stanford-page/home-page/today-hours.tsx

```diff
@@ -63,7 +63,7 @@ const LibrariesTodayHours = ({libraries, ...props}: {libraries: HoursProps["libr
           )
         }
         footer={
-          <div className="relative pb-100 md:rs-pb-7">
+          <div className="relative pb-[14rem] md:rs-pb-8">
```

Comment:
`pb-140` would do the same thing. As much as we can use regular TW classes, the better.